driver/CMakeLists.txt

```diff
@@ -51,6 +51,8 @@ add_executable(MIOpenDriver
     dm_softmax.cpp
     dm_sum.cpp
     dm_tensorop.cpp
+    dm_unfold.cpp
+    dm_fold.cpp
```

Comment:
plz sort

Reply:
resolve in [e76095e](https://github.com/moreh-dev/MIOpen/pull/37/commits/e76095e8fc1771a82978af13018b538b396877d1)